main.py

```diff
@@ -114,12 +114,14 @@ def fromCar():
                 lastRead[sensor] = req_body[sensor]
         if len(buffer) > (15*12) : #check buffer size and if it is greater than threshold
             writeToFireBase()
-            countdownToBufferClear.cancel()
+            countdownToBufferClear._stop()
+            countdownToBufferClear._delete()
```

Comment:
It's not ideal to be using the methods starting with underscores since they're psuedo-private, but it's better than it not working.